ozkour-front/tests/unit/TestDatePicker.spec.js

```diff
@@ -1,84 +1,110 @@
-import { mount } from "@vue/test-utils";
+import { mount, shallowMount } from "@vue/test-utils";
 import ChoosingDate from "@/components/ChoosingDate.vue";
 import { createTestingPinia } from "@pinia/testing";
 
 describe("ChoosingDate Component", () => {
-  it("Select Start Date from input", async () => {
-    const wrapper = mount(ChoosingDate, {
-      global: {
-        plugins: [createTestingPinia()],
-      },
-    });
-    const input = wrapper.find('[id="start"]');
-    await input.setValue("2021-01-01");
-
-    expect(input.element.value).toBe("2021-01-01");
-  })
+  describe("week day estimation", () => {
+    it("should return 1 when seekNextMonday is called and we are sunday", async () => {
```

Comment:
it("should return 1 when howManyDaysUntilNextMonday is called and we are sunday", async () => {